src/app/package-json.js

```diff
@@ -268,6 +272,8 @@ const packageJSON = (current, context) => {
 
   // Support the documentation tooling option.
   if (context.docs) {
+    result.files.push('docs');
```

Comment:
Because it makes the npm tarball self contained and makes the docs for that version available locally and offline.
Also, there's really no reason to *not* add it to npm.